src/OrchardCore.Modules/OrchardCore.Localization/Models/TranslationsDocument.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace OrchardCore.Localization.Models
+{
+    public class TranslationsDocument
```

Comment:
Will need to inherit from `Document` if intended to be stored and kept in sync in our multi level cache, e.g to be shared and kept in sync across multiple running instances, see the `TemplatesDocument` as an example